v1/bi/na/binary-data

```diff
@@ -2,11 +2,20 @@
   "category": "parsers",
   "contact": "dylan-lang@googlegroups.com",
   "description": "A DSL for parsing and assembling binary data",
-  "keywords": [],
+  "keywords": ["binary", "parser"],
   "name": "binary-data",
   "releases": [{
                  "dependencies": [],
                  "deps": [],
+                 "dev-dependencies": ["testworks", "sphinx-extensions"],
+                 "license": "MIT",
+                 "url": "A DSL for parsing and assembling binary data",
```

Comment:
Looks like this was meant to be the `"description"` field and you need a `"url"` field.

Was this a hand edit or a bug in `dylan publish`?

Reply:
It's my fault. 'Description' field it's duplicated in 'url' field.